priv/app/call_tracer.jsx

```diff
@@ -58,7 +58,8 @@ class CallsTable extends React.Component {
   onClick(id, event) {
     event.preventDefault();
 
-    const newOrder = (this.state.order === "desc") ? "asc" : "desc";
+    const changeColumn = (this.state.sortby === id);
```

Comment:
nonblocking: `changeColumn` is a bit misleading as it is true when the column did not change. maybe `isActive` is a better name (and used lower for the same thing)